src/Sulu/Bundle/MediaBundle/DependencyInjection/Configuration.php

```diff
@@ -96,6 +96,7 @@ public function getConfigTreeBuilder()
                             'Expires' => '+1 month',
                             'Pragma' => 'public',
                             'Cache-Control' => 'public',
+                            'X-SULU-MEDIA-CACHE' => 'MISS',
```

Comment:
What is this header doing? Doesn't that always render the same header?

Reply:
Its only set in PHP so when the image is generated, nginx will not set this header.

Reply:
But why is it needed?

Reply:
To see if the nginx correctly return the images and not always php is called and images are regenerated.

Reply:
Okay, totally makes sense :+1: